src/main/webapp/dev/js/common/instructorFeedbackResults.es6

```diff
@@ -319,6 +319,12 @@ function prepareInstructorFeedbackResultsPage() {
         });
     };
 
+    // decrease top-margin of expand button, if status message is present
+    if ($('#statusMessagesToUser').html().search('div') > 0) {
+        $('#expand-collapse-button').css('margin-top', '5px');
+        $('#expand-collapse-button + br').remove();
+    }
```

Comment:
There must be a way to use CSS (or JSP in the worst case) instead of JavaScript to achieve this.